### ISSUE TITLE
Integrated Gradients - viz tools 

### ISSUE THREAD

**Issue by innat** [keras-team/keras-cv#229]
> Visualize an average of the gradients along the construction of the input towards the decision

- https://github.com/sicara/tf-explain#integrated-gradients
- https://keras.io/examples/vision/integrated_gradients/

**Comment by LukeWood** [comment]
Thanks!  Great idea.  Added to the hotlist
